config/ftbquests/quests/chapters/refined_storage.snbt

```diff
@@ -1146,58 +1401,90 @@
 			}]
 		}
 		{
-			x: 6.0d
-			y: -7.5d
-			dependencies: ["6B04FC81351CD1AB"]
+			x: 5.5d
+			y: -0.5d
+			shape: "circle"
+			description: [
+				"\"I need more RANGE!!!\""
+				""
+				"Well, slap one of these bad boys in your Wireless Transmitter and you'll be able to access your network storage with infinite range."
+				""
+				"This, however, does not extend to other dimensions....."
+			]
+			dependencies: ["2E2811D1A0F2A492"]
 			id: "3E0D29502690E02F"
-			tasks: [
+			tasks: [{
+				id: "4DD51C8F38808EB9"
+				type: "item"
+				item: "rsinfinitybooster:infinity_card"
+			}]
+			rewards: [
 				{
-					id: "4DD51C8F38808EB9"
-					type: "item"
-					item: "rsinfinitybooster:infinity_card"
+					id: "020CFD2156077264"
+					type: "random"
+					table_id: 3567941291661635734L
 				}
 				{
-					id: "3C3F1953DAA71996"
-					type: "item"
-					item: "rsinfinitybooster:dimension_card"
+					id: "51EB8245F9BF2EBB"
+					type: "random"
+					table_id: 407746579787184593L
 				}
 			]
-			rewards: [{
-				id: "020CFD2156077264"
-				type: "random"
-				table_id: 3567941291661635734L
-			}]
 		}
 		{
-			x: 0.0d
-			y: -12.0d
+			x: -1.5d
+			y: -1.0d
 			shape: "rsquare"
-			description: ["A Storage Disk is an item where items from the storage network are stored on. It has to be put in a Disk Drive. The Storage Disk won’t despawn when dropped in the world."]
-			hide_dependency_lines: true
-			dependencies: ["01F18150EC923482"]
+			description: [
+				"It's time to create the \"Hard Drives\" of Refined Storage. To do this, we'll need a &9Storage Housing&r that we'll combine with a &aStorage Part&r to create a &dStorage Disk&r. Just simply make the desired size of part, then combine with the housing to create a disk."
+				""
+				"The Storage Disk is used to store your items virtually once placed inside of the Disk Drive. It has to be put in a Disk Drive. The Storage Disk won’t despawn when dropped in the world."
+				""
+				"&l&nIMPORTANT&r: Do not store NBT items on a disk. This increases overall lag in the chunk, and &lcan eventually corrupt your save&r. If you are unsure what NBT items are, the general rule of thumb is \"If it doesn't stack, don't put it in your system.\""
```

Comment:
I'm not too sure on the technical details, but everything I've heard surrounding RS has always pointed towards the amount of NBT items within an RS system because of how much data is stored/accessed within the chunk. The chunk ends up corrupting and usually causes most people to lose the worlds they have, but I'm not an expert on any of it by any means. This was the final solution I've heard to combat those issues.